README.md

```diff
@@ -63,16 +63,15 @@ npm install @google-cloud/functions
 // eslint-disable-next-line node/no-missing-require
 const {CloudFunctionsServiceClient} = require('@google-cloud/functions');
 
-// TODO(developer): replace with your prefered project ID.
-// const projectId = 'my-project'
-
 // Creates a client
 const client = new CloudFunctionsServiceClient();
+// TODO(developer): optionally replace with your prefered project ID.
+const projectId = await client.getProjectId();
 
-// project = 'my-project/*/locations/*' // Get the functions for a project.
+const parent = `projects/${projectId}/locations/-`; // Get all the functions for a project.
```

Comment:
Thank you for catching this! If you wouldn't mind leaving this as 'project', but changing the format as you described, that would be awesome!

Reply:
You want to me to reuse the `project` variable to change it from being the project id to the parent resource string? Or do you just want me to change `projectId` to `project`?

Reply:
Ah, here's what I would recommend: 

```
// TODO(developer): replace with your prefered project ID.
// const projectId = 'my-project'

const parent = `projects/${projectId}/locations/-`; // Get all the functions for a project.


async function listFunctions() {
  const [functions] = await client.listFunctions({
    parent,
    pageSize: page_size,
    pageToken: page_token,
  });

```